conf/st2.conf.sample

```diff
@@ -312,11 +312,13 @@ ssh_key_file = /home/stanley/.ssh/stanley_rsa
 # Default system user.
 user = stanley
 
-[timer]
-# Specify to enable Timer.
+# Old config group is timer and you can specify the exact same settings in [timer] section
+[timersengine]
+# Specify to enable Timer
 enable = True
 # Timezone pertaining to the location where st2 is run.
-local_timezone = America/Los_Angeles
+local_tz = America/Los_Angeles
```

Comment:
`local_tz` vs `local_timezone` ?

Reply:
It was auto-generated but I'll look into why the auto-gen code did this.